cartographer_ros/cartographer_ros/node.cc

```diff
@@ -441,14 +448,30 @@ void Node::StartTrajectoryWithDefaultTopics(const TrajectoryOptions& options) {
   AddTrajectory(options, DefaultSensorTopics());
 }
 
-std::unordered_set<std::string> Node::ComputeDefaultTopics(
-    const TrajectoryOptions& options) {
-  carto::common::MutexLocker lock(&mutex_);
-  return ComputeExpectedTopics(options, DefaultSensorTopics());
+std::vector<
+    std::set<cartographer::mapping::TrajectoryBuilderInterface::SensorId>>
+Node::ComputeDefaultSensorIdsForMultipleTrajectories(
+    const std::vector<TrajectoryOptions>& options) const {
+  using SensorId = cartographer::mapping::TrajectoryBuilderInterface::SensorId;
+  std::vector<std::set<SensorId>> trajectories_sensor_ids;
+  for (size_t i = 0; i < options.size(); ++i) {
+    std::string prefix;
+    if (options.size() > 0) {
+      prefix = "trajectory_" + std::to_string(i + 1) + "_";
```

Comment:
Same as above, change `trajectory_` to `bag_`?

Also, maybe adding a slash at the end to put it in a namespace (`"_"` to `"/"`) might be more in the spirit of ROS.

Reply:
Made the change to bag_.
I chose `_` because this is consistent with the current behavior.
Does `/` any logic to make the remapping easier? If yes, could you please point me to documentation?